Makefile-deps.mk

```diff
@@ -16,6 +16,7 @@ PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
 TOOLS_DIR := $(PROJECT_DIR)/hack/internal/tools
 BIN_DIR ?= $(PROJECT_DIR)/bin
 EXTERNAL_CRDS_DIR ?= $(PROJECT_DIR)/dep-crds
+EXTERNAL_MANIFESTS_DIR ?= $(PROJECT_DIR)/dep-manifests
```

Comment:
And this?

Reply:
yep, you are right, artifacts from previous approach